src/node_worker.cc

```diff
@@ -254,6 +255,7 @@ void Worker::Run() {
         env_->ProcessCliArgs(std::vector<std::string>{},
                              std::move(exec_argv_));
       }
+      if (is_stopped()) return;
```

Comment:
I think this needs to be inside the following locking scope, otherwise we release and-relock the mutex and leave a window for race conditions? (When using `is_stopped()` that would conflict with the mutex added above, so I think it would be just `if (stopped_) return;` in that case)

Reply:
sure.